src/Ionide.KeepAChangelog/Library.fs

```diff
@@ -4,55 +4,40 @@ module Domain =
     open SemVersion
     open System
 
-    // TODO: a changelog entry may have a description?
     type ChangelogData =
-        { Added: string list
-          Changed: string list
-          Deprecated: string list
-          Removed: string list
-          Fixed: string list
-          Security: string list
-          Custom: Map<string, string list>}
+        {
+            Added: string
+            Changed: string
+            Deprecated: string
+            Removed: string 
+            Fixed: string
+            Security: string
+            Custom: Map<string, string>
+        }
         static member Default =
-            { Added = []
-              Changed = []
-              Deprecated = []
-              Removed = []
-              Fixed = []
-              Security = []
+            { Added = String.Empty
+              Changed = String.Empty
+              Deprecated = String.Empty
+              Removed = String.Empty
+              Fixed = String.Empty
+              Security = String.Empty
               Custom = Map.empty }
 
         member this.ToMarkdown () =
-
-            let renderItems (items : string list) =
-                items
-                |> List.map (fun item ->
-                    "* " + item
-                )
-                |> String.concat Environment.NewLine
-
-            let section name items =
-                match items with
-                | [] -> []
-                | items ->
-                    $"### {name}"
-                    + Environment.NewLine
-                    + Environment.NewLine
-                    + (renderItems items)
-                    + Environment.NewLine
-                    |> List.singleton
+            let section name (body: string) =
```

Comment:
I think I didn't use interpolation on purpose because .NET Framework MSBuild running in VS didn't have access to the feature. Has that changed?

Reply:
Actually, I think it is already used in other places:
https://github.com/ionide/KeepAChangelog/blob/8c9c5fa8550c6ea977aa8d4d2cd85e353190142c/src/Ionide.KeepAChangelog.Tasks/Library.fs#L53

Reply:
Should be fine then?

Reply:
Good catch - so we're at least not _worse_ than before, and if it is a big problem we'd have to fix all occurrences of it. Thanks for checking!